packages/react/src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,116 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import { UIComponent, commonPropTypes } from '../../lib'
+import { ReactProps } from '../../types'
+import FlexItem from './FlexItem'
+
+export type FlexGap = 'gap.small' | 'gap.medium' | 'gap.large'
+export type FlexPadding = 'padding.medium'
```

Comment:
Why we have just one option here. Moreover why is the value medium? It doesn't make sense without any other values in my opinion.

Reply:
agree  - but lets refine the set of `padding` values further down the road. What I am not sure at the moment is whether `padding` should be specified this way, as it is a 2D thing - what if, for instance, we would need to make vertical padding lesser than horizontal one, as it is usually the case for button? `medium` name was selected just to be consistent with the dictionary we've used so far for size values.

With that being said, I acknowledge the issue, but it was intentionally left visible, as the approach to organise these values (by putting them under general design schema) needs to be introduced as a follow-up work.